doc/python_api/rst/bge_types/bge.types.KX_Scene.rst

```diff
@@ -143,10 +143,15 @@ base class --- :class:`PyObjectPlus`
 
       :type: Vector((gx, gy, gz))
 
-   .. attribute:: lodGlobalScale
+   .. method:: setLodScale(pythonObjectList, scale)
```

Comment:
Because I guess it is slow. For i in range 1000: listOfTrees[i].obj.lodManager.distanceScale = scale is slow in python. I guess this is faster in c++.